enterprise/api/v1/urls.py

```diff
@@ -24,3 +24,8 @@
     'enterprise-customer-entitlement',
 )
 router.register("catalogs", views.EnterpriseCourseCatalogViewSet, 'catalogs')
+router.register(
+    "enterprise-customer-reporting",
+    views.EnterpriseCustomerReportingConfigurationViewSet,
+    'enterprise-customer-reporting',
```

Comment:
Since we're going to be doing underscores anyway, we should probably go ahead and do it here too.